rollout/trafficrouting_test.go

```diff
@@ -318,11 +326,7 @@ func TestRolloutWithExperimentStep(t *testing.T) {
 			return nil
 		})
 		f.fakeTrafficRouting.On("SetHeaderRoute", mock.Anything, mock.Anything).Return(nil)
-		f.fakeTrafficRouting.On("VerifyWeight", mock.Anything).Return(func(desiredWeight int32, weightDestinations ...v1alpha1.WeightDestination) error {
```

Comment:
because these assertions are already present here : https://github.com/argoproj/argo-rollouts/pull/2734/files#:~:text=with%20correct%20value-,assert.Equal(t%2C%20int32(10)%2C%20desiredWeight),-assert.Len